core/validatorapi/teku.go

```diff
@@ -43,10 +43,15 @@ type TekuProposerConfigProvider interface {
 }
 
 func (c Component) TekuProposerConfig(ctx context.Context) (TekuProposerConfigResponse, error) {
+	var defaultFeeRecipientAddr string
+	for _, addr := range c.feeRecipientAddressByPubkey {
+		defaultFeeRecipientAddr = addr
+		break
+	}
 	resp := TekuProposerConfigResponse{
 		Proposers: make(map[string]TekuProposerConfig),
 		Default: TekuProposerConfig{ // Default doesn't make sense, disable for now.
```

Comment:
as the comment says, "Default doesn't make sense, disable for now", rather just disable and use zero values for the rest.